### PR TITLE
Vdh razor spikes

### PR DESCRIPTION
**Pull request by yajinni**
https://www.warcraftlogs.com/reports/rz6WxLbAmTgnFXQP/#fight=3&source=3

image

### CODE REVIEW

File: src/Parser/DemonHunter/Vengeance/Modules/Talent/RazorSpikes.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//WCL: https://www.warcraftlogs.com/reports/rz6WxLbAmTgnFXQP/#fight=3&source=3
+
+  damage = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.RAZOR_SPIKES_TALENT.id);
+  }
+
+  on_toPlayer_damage(event) {
```
Comment (Gebuz) [inline src/Parser/DemonHunter/Vengeance/Modules/Talent/RazorSpikes.js]: should be on_**by**player_damage.

File: src/Parser/DemonHunter/Vengeance/Modules/Talent/RazorSpikes.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'Parser/Core/Analyzer';
+import { formatNumber } from 'common/format';
+import SCHOOLS from 'common/MAGIC_SCHOOLS';
+import calculateEffectiveDamage from 'Parser/Core/calculateEffectiveDamage';
+
+const PHYSICAL_DAMAGE_INCREASE = 0.20;
```
Comment (Gebuz) [inline src/Parser/DemonHunter/Vengeance/Modules/Talent/RazorSpikes.js]: should be 0.15

**Comment by yajinni** [comment]
Woah idk how i messed that up. Nice catch, thanks! Made the changes.

File: src/Parser/DemonHunter/Vengeance/Modules/Talents/RazorSpikes.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+
+import SPELLS from 'common/SPELLS/index';
```
Comment (Gebuz) [inline src/Parser/DemonHunter/Vengeance/Modules/Talents/RazorSpikes.js]: `/index` is redundant here

**Pull request merged**